locales/fr/json.json

```diff
@@ -173,10 +173,10 @@
     "Choose Type": "Choisir le type",
     "Christmas Island": "Île Christmas",
     "City": "Ville",
-    "Click here to re-send the verification email.": "Cliquer ici pour renvoyer l'e-mail de vérification.",
-    "click here to request another": "cliquer ici pour en demander un autre",
-    "Click here to reset your password:": "Cliquer ici pour réinitialiser votre mot de passe :",
-    "Click to choose": "Cliquer pour choisir",
+    "Click here to re-send the verification email.": "Cliquez ici pour renvoyer l'e-mail de vérification.",
+    "click here to request another": "Cliquez ici pour en demander un autre",
```

Comment:
il faudrait un petit "c" pour rester cohérent

Reply:
Tout le reste du document est à l'infinitif, autant y rester

Reply:
> il faudrait un petit "c" pour rester cohérent
It seems to be a typo on the original source code.
Button labels are all capitalized but this one - which looks weird UI-wise.

> Tout le reste du document est à l'infinitif, autant y rester
The "Click her to" refers to an indication/directive ("Please click her to -do something-) whereas other entries refer to the action itself (ie: "Add Team Member", "Do something"). In french, we can do the nuance, hence my edit.
In fact, I even forgot to change the two lines "Drop file or click to choose" which should be "Déposez le fichier ou cliquez pour choisir" on the same logic.

----
> il faudrait un petit "c" pour rester cohérent
Il s'agit d'une typo sur le code source d'origine.
Les labels de boutons sont tous en majuscule sauf celui-ci, ce qui semble étrange en terme d'UI.

> Tout le reste du document est à l'infinitif, autant y rester
La mention "Click here to" se réfère à une indication/directive, ("Veuillez cliquer ici pour - faire qqch -"), là ou les autres infinitifs se réfèrent plus à l'action en question (ie: "Add Team Member" et non "Click here to add a team member"). En français on peut faire la nuance, d'où ma suggestion de correction.
En fait, j'ai même oublié de traduire les deux lignes "Drop file or click to choose" qui devraient être "Déposez le fichier ou cliquez pour choisir" sur la même logique.

Reply:
ce n'est pas le premier terme en majuscule et minuscule, c'est le choix des développeurs, pas le notre, donc il faut le c en minuscule

Pour l'infinitif, tout est sujet à interprétation, les exemples sur les "Click here" montrent cette complexité de traduction justement. Je n'ai rien contre l'une ou l'autre des versions, mais effectivement en relisant plus longuement le fichier, on a bien de l'infinitif et de l'indicatif.

I'm ok with your translation, but please fix the mini c ;)

Reply:
@caouecs En effet, je repasser le "c" en minuscule : après inspection détaillée [de la source](https://github.com/laravel/ui/blob/master/src/Auth/bootstrap-stubs/auth/verify.stub), le bouton est inclus dans une phrase, et le minuscule fait sens.
Sauf indication contraire, je vais aussi passer les deux lignes "Drop file or click to choose" à l'impératif, sur la même base logique qu'évoqué ci-dessus.
https://github.com/Laravel-Lang/lang/blob/26f113d15c378cf13c9e45f28fcddbba2c0b94b9/locales/fr/json.json#L267-L268

Reply:
Laravel Nova 4.22.1 removed these two lines: https://github.com/Laravel-Lang/lang/commit/30b6744aa1bd70c2be5e4f8f1a10158055ffad7f

Reply:
Sorry for the trigger. This has been removed duplicate keys 😅